saclettre.c

```diff
@@ -1,140 +1,200 @@
 #include <stdio.h>
+#define MAX_LETTRE 27
+#define TOTAL_JETON 102
 
-/* Essai : ce programme initialise et affiche un tableau contenant l'ensemble des lettres du 
-sac de lettres, leur quantité et le nombre de points qui leur sont associés*/
-#define MAX_LETTRE 26
-#define MAX_JETON 27
 
+/* Déclaration des types globaux*/
+
+// type lettre
+struct lettre
+{
+  int id_lettre;
+  int nbpoint, quantite;
+};
+
+
+
+// variables globales
+struct lettre tablettre[MAX_LETTRE-1];
+char tabjeton[TOTAL_JETON - 1];
```

Comment:
Shouldn't it be 'MAX_LETTRE' instead of 'MAX_LETTRE - 1' ? :)
Same for TOTAL_JETON